botocore/signers.py

```diff
@@ -297,6 +303,8 @@ def get_auth_instance(
             return auth
 
         credentials = request_credentials or self._credentials
+        if credentials and (cred_method := credentials.method):
```

Comment:
[very non-blocking, mega-nit]: is there a reason this is in parenthesis?  It makes it -slightly- harder to read for me

Reply:
Also as a note, this is potentially breaking if credentials doesn't have a method if someone is doing something hackey and using duck-typing to pass in a different credentials class.  This is a private interface though, so I think that's okay?